lib/util/xml.ts

```diff
@@ -5,8 +5,6 @@ import * as xml2js from "xml2js";
 
 export function stringifyXML(obj: any, opts?: { rootName?: string }) {
   const builder = new xml2js.Builder({
-    explicitArray: false,
-    explicitCharkey: false,
```

Comment:
In `@azure/core-http` which is basically a copy of this project + all changes we wanted to make in Track 2, I see these two properties in use. Are we sure that the Builder does not need these?

Reply:
I see both of them in https://github.com/Leonidas-from-XIV/node-xml2js#options

Reply:
they are part of the parser options not the builder options and these are set at lines 18 and 19 here.

Reply:
Ah ok